### PR TITLE
Justify post content (issue #831: Text in post is not justified)

### LINKED ISSUE #813: Text in post is not justified
## Expected Behavior
Content text in post page should be justified.

## Current Behavior
Content text in post page is not justified.

## Possible Solution

Modifying css should be enough.

## Steps to Reproduce

1. Visit https://hugo-toha.github.io/posts/markdown-sample/ and scroll down.

### Screenshots

image

### PR DESCRIPTION
**Pull request by BernatBC**
### Issue

Fixes #813.

### Description

Had to justify `p` element by default cause in some cases the about section also had problems. Had to center the date displayed in posts. There's probably a more neat way to do it.

### Test Evidence

image

You can also visit [bernatbc.tk/posts/](https://bernatbc.tk/posts/) and select any post.

### CODE REVIEW

File: assets/styles/components/texts.scss [hunk 1]
```diff
@@ -80,6 +82,8 @@ html[data-theme='dark'] {
 
   p {
     color: get-dark-color('text-color');
+    text-align: justify;
+    text-justify: inter-word;
```
Comment (hossainemruz) [inline assets/styles/components/texts.scss:86]: You don't need to repeat these lines. Only add CSS that should be changed for dark mode.

File: layouts/_default/single.html [hunk 2]
```diff
@@ -39,7 +39,7 @@
         <div class="author-profile ml-auto align-self-lg-center">
           <img class="rounded-circle" src='{{ partial "helpers/get-author-image.html" . }}' alt="Author Image">
           <h5 class="author-name">{{ partial "helpers/get-author-name.html" . }}</h5>
-          <p class="text-muted">{{ .Page.Date | time.Format ":date_full" }}{{ if site.Params.features.readingTime }} | {{ .ReadingTime }} minute{{ if (ne .ReadingTime 1) }}s{{ end }}{{ end }}</p>
+          <p class="text-muted" style="text-align: center;">{{ .Page.Date | time.Format ":date_full" }}{{ if site.Params.features.readingTime }} | {{ .ReadingTime }} minute{{ if (ne .ReadingTime 1) }}s{{ end }}{{ end }}</p>
```
Comment (hossainemruz) [inline layouts/_default/single.html:42]: Please don't use inline css. Just add a `p` element under `author-profile` and apply CSS there:
https://github.com/hugo-toha/toha/blob/5b31653ed24d1607edbfe0bb598a381b72bf4deb/assets/styles/layouts/single.scss#L42

**Review by hossainemruz — CHANGES_REQUESTED** [review]
Thank you for the PR @BernatBC. Left few comments.

File: assets/styles/components/texts.scss (hunk 1 above)
Comment (BernatBC) [reply assets/styles/components/texts.scss]: Ok, thank you for this advice. Now it's fixed.

File: layouts/_default/single.html (hunk 2 above)
Comment (BernatBC) [reply layouts/_default/single.html]: Ok, now I added the text-align into the css file.

**Review by hossainemruz — APPROVED** [review]
LGTM 🚀

**Pull request merged**